scripts/release.sh

```diff
@@ -25,6 +25,14 @@ git add CHANGELOG.md package.json
 git commit -m "Prepare version $1"
 git tag $1
 yarn publish ./build/dist/sonar-ui-common-v$1.tgz
+
+if [ $? -gt 0 ]
+  then
+    echo "Publish failed, aborting"
+    npmrc default
```

Comment:
Should we also revert the commit and tag ?

Reply:
Good idea! I see now that it really messes stuff up if you run it again afterwards 😑 .